core/feconf.py

```diff
@@ -972,6 +972,7 @@ def get_empty_ratings() -> Dict[str, int]:
 LIBRARY_TOP_RATED_URL = '/community-library/top-rated'
 MACHINE_TRANSLATION_DATA_URL = '/machine_translated_state_texts_handler'
 MERGE_SKILLS_URL = '/merge_skills_handler'
+METADATA_VERSION_HISTORY_URL_PREFIX = '/version_history/metadata'
```

Comment:
These seem like json urls so let's follow the structure of other such urls (i.e. include "handler").

Reply:
Don't just wait, you should probably escalate. I'm not aware of any flakes at the moment, so I'm not sure if the dev workflow team is looking into this. 

@gp201 @U8NWXD can probably give you an update.

Reply:
There are lots of flakes that just got introduced. We're fixing them right now in #15732

Reply:
Thanks for the update @U8NWXD .

Reply:
@seanlip I have pushed the code for now using `--no-verify` and also attaching the screenshot of the error just as a proof that this was not caused by my PR.


![Screenshot from 2022-07-16 23-56-24](https://user-images.githubusercontent.com/70739349/179367655-b2a41cc3-d79c-4e36-acc6-993c58b68a87.png)